wagtailio/core/migrations/0061_remove_excess_tables.py

```diff
@@ -0,0 +1,25 @@
+# Generated by Django 5.1.5 on 2025-01-24 22:38
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+    """
+    The Wagtail AB Testing module as well as the Google import module were removed from this project without dropping its tables.
+    """
+
+    dependencies = [
+        ("core", "0060_alter_contentpage_body_alter_homepage_body"),
+    ]
+
+    operations = [
+        migrations.RunSQL(
+            elidable=True,  # This migration can be removed during a squash
```

Comment:
Yay! Suggesting this flag is a comment I often make during reviews of manual migrations like this one - but you are already way ahead here 🥇